Makefile

```diff
@@ -49,14 +53,26 @@ apply-single: ## Apply terraform for a project, make apply-single project=<proje
 destroy-single: ## Destroy terraform for a project, make destroy-single project=<project name>
 	. ./setup.sh -d ${project}
 
-.PHONY: destroy
-destroy: ## Destroy stack
-	. ./setup.sh -d
+.PHONY: init-list
+init-list: ## Init terraform for a list of projects, make init-list list="<comma separated list of projects>"
```

Comment:
Any chance we could put the make list stuff in as a separate PR just for ease of keeping this PR small? I haven't had a chance to look at that new code/test it

Reply:
Sure, will create a new PR for this